pkgs/desktops/surf-display/default.nix

```diff
@@ -53,5 +53,6 @@ stdenv.mkDerivation rec {
     maintainers = with maintainers; [ etu ];
     license = licenses.gpl2;
     platforms = platforms.linux;
+    broken = true;
```

Comment:
Surf display is not broken on master: https://hydra.nixos.org/jobset/nixpkgs/trunk/jobs-tab?filter=surf-display